src/Functions/README.md

```diff
@@ -65,7 +65,7 @@ directive:
 ```
 
 ``` yaml
-branch: powershell-function
+branch: main
```

Comment:
@Francisco-Gamino There are 2000+ generated files get updated because of this PR (see #18209). Does it make sense to you? Is swagger of main branch that different from powershell-function branch?

Another thing, I'm going to put "Fixed an issue that `New-AzFunctionApp` cmdlet should write a warning message when setting default values for parameters that are not provided." into changelog. Let me you if you want to add anything else.

Reply:
Hello @isra-fel -- No, that does not make sense. Switching from one branch to another should not generate these many files. What I think it could be happening here is that more Swagger definitions have been added to the API version 2019-08-01 that I am using, so I will need to remove the extra cmdlets that get generated. The Swagger files from the `powershell-function` branch are at least two years old and have not been updated. 

Should we revert back to the `powershell-function`? After this release I can work on removing the extra generated cmdlets to move to the `main` branch. Please advice. Thank you. 

/cc @wyunchi-ms